composer.json

```diff
@@ -9,14 +9,31 @@
             "EventSourcingWorkshop\\": "src"
         }
     },
+    "autoload-dev": {
+        "psr-4": {
+            "EventSourcingWorkshopTest\\": "test"
+        }
+    },
     "require": {
```

Comment:
Probably need `ext-pdo_sqlite` here.

After much thinking, adding MySQL to the mix would just be a pain, at this point.

Reply:
a9e22660715d6d53eaf372b78796700299ad4c35